package.json

```diff
@@ -7,7 +7,7 @@
     "build": "babel src  --out-dir production",
     "start": "node production/src/index.js",
     "dev": "nodemon --exec babel-node index.js",
-    "test": "nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit"
+    "test": "export NODE_ENV=test &&  sequelize db:migrate:undo:all  && sequelize db:migrate && nyc --reporter=html --reporter=text mocha  --require  @babel/register  src/tests/*.js --exit"
```

Comment:
Put `sequelize db:migrate` in a separate script and reference that instead. Do the same for the undo command. 

Instead of running it in sequence here, you can call the scripts individually under Travis `before_script`